packages/eslint-plugin/docs/rules/strict-enums.md

```diff
@@ -0,0 +1,149 @@
+# `strict-enums`
+
+Disallows the usage of unsafe enum patterns.
+
+## Rule Details
+
+Horrifyingly, the TypeScript compiler will allow you to set any number to a variable containing a number enum, like this:
```

Comment:
> Horrifyingly

We try not to express such poignant emotions in our docs. Even if we may sometimes feel this way on the inside 😄 

```suggestion
The TypeScript compiler will allow you to set any number to a variable containing a number enum, like this:
```